shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -23,13 +25,20 @@ use std::net::SocketAddr;
 use std::sync::Arc;
 use std::time::Duration;
 use tokio::sync::{mpsc, oneshot, RwLock};
+use uuid::Uuid;
+use version_compare::Cmp;
 
 mod node;
 
 #[derive(Deserialize, Debug, Clone)]
 pub struct CassandraSinkClusterConfig {
+    /// contact points must be within the specified data_center and rack.
```

Comment:
Would it be possible to emit a warning if this occurs?

Reply:
there is a warning! :D

Reply:
`racing::warn!("A contact point node is not in the configured rack, this node will receive traffic from outside of its rack");`

Reply:
Ah I see, missed that. Thanks